src/features/game/components/Decorations.tsx

```diff
@@ -673,6 +674,26 @@ export const Decorations: React.FC<Props> = ({ state }) => (
       />
     )}
 
+    {state.inventory["Wicker Man"] && (
+      <img
+        style={{
+          width: `${GRID_WIDTH_PX * 1.2}px`,
```

Comment:
To get the exact width you can multiply the image file width (19px) by the `PIXEL_SCALE` constant.

E.g. width: `${19 * PIXEL_SCALE}px`,